### DISCUSSION TITLE
I'm sorry if this is unrelated, but I have an idea that I think would be very useful.

### DISCUSSION THREAD

**Discussion by memesong** [ScratchAddons/ScratchAddons#5174] [Ideas]
Maybe there could be a version of Scratch Addons for Snap!, another block coding website similar to Scratch. I really don't like the aesthetic of Snap!'s blocks, and I think if there were a way to make Snap! look like Scratch, then that would help get me into coding with Snap!. If this can't be done or it's something that needs to be requested elsewhere, then I understand. Otherwise, I'm really excited to see what would become of Snap! Extensions.

**Comment by mxmou** [comment]
I think forking the source code might be better than making a browser extension because most Snap<i>!</i> features don't need access to the Snap<i>!</i> website to work. Cloud saving is the only exception.

**Reply by memesong** [reply]
Do you mean using the Snap! source code to create a different editor entirely, or am I misunderstanding what you're saying?

**Reply by BroJac5246** [reply]
> Do you mean using the Snap! source code to create a different editor entirely, or am I misunderstanding what you're saying?

Yes. Think of it as the equivalent of Scratch's remixing feature.

**Comment by BroJac5246** [comment]
I doubt we would create a version of Scratch Addons for Snap! in the near future (MV3 is our current priority), but I still like the idea. As mxmou pointed out, forking Snap! might be the best solution, since many of SA's addons were designed with Scratch in mind. That said, a Snap! Addons could take inspiration from Scratch Addons and its features.

**Reply by memesong** [reply]
I don't exactly know what MV3 is, can you explain?

**Reply by BroJac5246** [reply]
MV3 is short for "Manifest Version 3." Basically, browser extensions give the browser data about themselves (such as their name and what code to run) in a "manifest." However, they're are multiple forms that a manifest can take. The current one is Manifest Version 2, but Chrome is about to require extensions to use MV3 instead. However, when a new manifest version is introduced, other limits and additions are imposed on browser extensions. This means a lot of work to go from one version to another- which Scratch Addons is about to have to do.

TL;DR: Chrome is about to update how extensions work, and Scratch Addons has a lot of work to get ready.
